.github/workflows/e2e-tests.yml

```diff
@@ -31,5 +31,11 @@ jobs:
           cd js
           npm install
 
+      - name: Install crane
+        run: |
+          VERSION="v0.19.1"
+          curl -LO https://github.com/google/go-containerregistry/releases/download/${VERSION}/go-containerregistry_Linux_x86_64.tar.gz
+          sudo tar -xvzf go-containerregistry_Linux_x86_64.tar.gz -C /usr/local/bin crane
```

Comment:
No need to do that, do something like [that](https://github.com/kubewarden/kubewarden-controller/blob/c6bf2e891f7a167c8afd5a7424fd8842b55228d2/.github/workflows/release.yml#L84)